manifests/composer.pp

```diff
@@ -28,6 +31,7 @@
   Stdlib::Absolutepath $path           = $php::params::composer_path,
   $proxy_type                          = undef,
   $proxy_server                        = undef,
+  $channel                             = 'stable',
```

Comment:
Please add a datatype for this. Would an enum make sense here?

Reply:
Hi Tim, i've added it as an enum now.

Reply:
looks good to me. only the test fixes are left.